explorer/rest/tests/test/DatabaseTestUtils.py

```diff
@@ -90,3 +100,16 @@ def initialize_database(db_config):
 			)
 
 		connection.commit()
+
+
+class DatabaseTestBase(unittest.TestCase):
+	@classmethod
+	def setUpClass(cls):
+		cls.postgresql = testing.postgresql.Postgresql()
+		cls.db_config = DatabaseConfig(**cls.postgresql.dsn(), password='')
+		cls.network = 'MAINNET'
```

Comment:
why this ?

Reply:
I added new parameters (network) in the `NemRestFacade` and `NemDatabase` classes.

The network mainly use for convert `public_key_to_address`

Reply:
my main goal is to share `network` properties with the subclass.